README.md

```diff
@@ -19,20 +19,30 @@ Documentation
 
 [SwiftBuild.docc](SwiftBuild.docc) contains additional technical documentation.
 
+To view the documentation in browser, run the following command at the root of the project:
+```bash
+docc preview SwiftBuild.docc
+```
+
+in MacOS, use:
```

Comment:
Just a quick heads up that the canonical spelling should be "macOS" (with a lowercase m).

Reply:
Thank you, I know better. 🤦